crates/sncast/src/helpers/braavos.rs

```diff
@@ -140,3 +140,6 @@ where
             .is_interactive(SignerInteractivityContext::Other)
     }
 }
+
+pub const BRAAVOS_DISABLED_MESSAGE: &str =
+    "Integration with Braavos accounts is currently disabled";
```

Comment:
note: This would be better as helper function with account type parameter, but we have two enums named `AccountType`  (will be resolved in https://github.com/foundry-rs/starknet-foundry/issues/2571).